README.md

```diff
@@ -1,8 +1,11 @@
-![Gini Vision Library for iOS](https://www.gini.net/assets/GiniVision_Logo.png)
+![Gini Vision Library for iOS](https://github.com/gini/gini-vision-lib-ios/blob/master/GiniVision_Logo.png?raw=true)
```

Comment:
So basically the code is referencing its own location on github? Why not just add the image to the repository?

Reply:
It's is in the repository actually. Yes, I can also set this "./GiniVision_logo.png" reference.